src/uproot/_dask.py

```diff
@@ -312,23 +321,30 @@ def _dask_array_from_map(
 
 
 class _UprootReadNumpy:
-    def __init__(self, ttrees, key) -> None:
+    def __init__(self, ttrees, key, interp_options) -> None:
         self.ttrees = ttrees
         self.key = key
+        self.interp_options = interp_options
 
     def __call__(self, i_start_stop):
         i, start, stop = i_start_stop
         return self.ttrees[i][self.key].array(
-            entry_start=start, entry_stop=stop, library="np"
+            entry_start=start,
+            entry_stop=stop,
+            library="np",
+            ak_add_doc=self.interp_options["ak_add_doc"],
```

Comment:
I think it's better to be explicit, to control the list of options. Then adding a new one would be a matter of searching for all instances of `ak_add_doc` and adding the new one next to that.

`**self.interp_options` passes everything in the `interp_options` dict through, which might be right or it might silently overshadow arguments of `TTree.arrays` that aren't `interp_options`. We shouldn't _create_ different types of arguments with the same names, but it would just be easier to catch such mistakes with an explicit pass-through.